src/aat/java/uk/gov/hmcts/reform/em/orchestrator/functional/AutomatedBundlingScenarios.java

```diff
@@ -55,6 +65,16 @@ public void testInvalidConfig() {
                 response.getBody().jsonPath().getString("errors[0]"));
     }
 
+    @Ignore
```

Comment:
For now, pls remove Ignored test cases. As we have to give unnecessary justification to release team why this test been ignored.